ledger-core/v2/runner/executor/builtin/builtin.go

```diff
@@ -86,12 +87,26 @@ func (b *Runner) CallMethod(
 	}
 	contract := b.CodeRegistry[contractName]
 
-	methodFunc, ok := contract.Methods[method]
+	methodFuncObject, ok := contract.Methods[method]
```

Comment:
If we rename it to methodObject we won't use something like return methodFunc.Func(data, args) (see above)

Reply:
sure